greenpeace/planet4-plugin-gutenberg-blocks
# PLANET-6425 Update Timeline.js

Ref: https://jira.greenpeace.org/browse/PLANET-6425
Upstream: https://github.com/NUKnightLab/TimelineJS3/issues/670

---

This fixes Timeline block for multilingual sites that fail with error:
```
[Deprecation] Synchronous XMLHttpRequest on the main thread is
deprecated because of its detrimental effects to the end user's
experience.
```

### Testing

1. Add this to `Makefile.include`:
```
NRO_NAME := mexico
NRO_DB_VERSION ?= latest
```
2. Run `make nro-enalbe`. This will get you Mexico website locally.
3. Browse at `/logros-greenpeace-mexico/`. There is a timeline block which is broken the way the ticket screenshot shows.
4. Switch to this branch and re-build assets.
5. The timeline block should work fine.
